frappe/core/doctype/background_report_result/test_background_report_result.py

```diff
@@ -5,6 +5,36 @@
 
 import frappe
 import unittest
+import json
+
 
 class TestBackgroundReportResult(unittest.TestCase):
-	pass
+
+	def setUp(self):
+		self.report = frappe.get_doc({
+			"doctype": "Report",
+			"name": "Permitted Documents For User"
+		})
+		self.filters = {
+			"user": "Administrator",
+			"doctype": "Role"
+		}
+		self.background_report_doc = frappe.get_doc({
+			"doctype": "Background Report Result",
+			"report_name": self.report.name,
+			"filters": json.dumps(self.filters),
+			"ref_report_doctype": self.report.name,
+			"report_type": self.report.report_type,
+			"query": self.report.query,
+			"module": self.report.module
+		}) \
```

Comment:
why . insert in next line?
add it to same line, it adds unnecessary new line

Reply:
Requested changes are done.